test/test_helper.rb

```diff
@@ -13,7 +13,7 @@
 
 # Start FakePuppetDB-Server
 server_thread = Thread.new do
-  Rack::Server.start(app: FakePuppetDB.new, Host: "localhost", Port: 8084)
+  Rack::Server.start(app: FakePuppetDB.new, Host: "localhost", Port: 8085)
```

Comment:
I had to change that because something else is running on 8084 on GitHub actions